src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -263,14 +284,14 @@ var ContainerSettingsComponent = React.createClass({
       <div>
         <div className="row">
           <div className="col-sm-6">
-            <FormGroupComponent
-              attribute="container.docker.image"
+            <StoreFormGroupComponent
+              fieldId="container.docker.image"
```

Comment:
Let's stick with fieldId and not try them to use as paths.
This here could be just:
```
dockerImage
```